doc/ref_arch/openstack/chapters/chapter05.md

```diff
@@ -18,10 +18,24 @@ This chapter presents a consolidated set of OpenStack Service APIs corresponding
 
 ### 5.2.1. Keystone
 
-| **OpenStack Service** | **Link for API and CLI** | **API/client Minimum (Baseline) Version** |
-|------------------|----------------------------------------------------|-------------------|
-|Identity: Keystone | https://docs.openstack.org/api-ref/identity/v3/index.html | Version 3.10 |
-|Identity: Keystone | https://docs.openstack.org/python-keystoneclient/latest/using-api-v3.html | Version 3.10 |
+| **OpenStack Service** | **Version** |
+|-----------------------|-------------|
+| Identity: Keystone    | 3.8         |
```

Comment:
Also fix the keystone version in §5.3.1

Reply:
Done. I would suggest to cleanup 5.3.1 when all APIs are well defined first (it's an aggregate)

Reply:
I agree. @karinesevilla if you agree I can open an issue.